deps/v8/src/snapshot/code-serializer.cc

```diff
@@ -55,10 +54,11 @@ ScriptCompiler::CachedData* CodeSerializer::Serialize(
   if (isolate->debug()->is_loaded()) return nullptr;
 
   // Serialize code object.
+  Handle<String> source(String::cast(script->source()), isolate);
   CodeSerializer cs(isolate, SerializedCodeData::SourceHash(source));
   DisallowHeapAllocation no_gc;
   cs.reference_map()->AddAttachedReference(*source);
-  ScriptData* script_data = cs.Serialize(info);
+  ScriptData* script_data = cs.SerializeSharedFunctionInfo(info);
 
```

Comment:
@hashseed RE https://github.com/nodejs/node/pull/20300#issuecomment-399522655. Can you double-check that this produces buffers with more data (after script execution) than say those with the legacy Node API (before execution).

Reply:
NM I goofed my test.